5.3/5.3.5/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 5.3.5
+Создать объект со следующей логикой:
+Запрашивает у пользователя любой текст: "Введите любой текст".
+А потом с помощью алерта выводит информацию об этом тексте (учитывай перенос строк в алерте), например:
+Введенный текст: "Абра ка табра";
+Всего символов: 13;
+Всего слов: 3.
+<script>
+	const infoObj = {
+		userText: '',
+		wordCount: 1,
+		characters: null,
+		finalMessage: '',
+		init() {
+			this.getMessage();
+			this.getNumberTextCharacters();
+			this.getWordCount();
+			this.formMessage();
+			this.showMessage();
+		},
+		getMessage() {
```

Comment:
Let's start a method name (or function) from `get` if the method returns something otherwise let's start from another verb. Here we can name the method like `askMessage`